src/views/product-landing/components/hero-heading-visual/index.tsx

```diff
@@ -7,13 +7,24 @@ function HeroHeadingVisual({
 	image,
 	productSlug,
 }: HeroHeadingVisualProps) {
+	const gradientDefault = {
```

Comment:
I thought it could be nice to extend this component to support a default. I could totally make a bespoke one though if that feels better 😸 the designs are _slightly_ different 🤔

Reply:
This seems v reasonable to me!